### PR TITLE
i have updated as per request.

### PR DESCRIPTION
**Pull request by MicroDreamIT**
Please take a look sir.

### CODE REVIEW

**Comment by MicroDreamIT** [comment]
I have done as per request, what should i do now?

File: src/apps.json [hunk 1]
```diff
@@ -7154,12 +7154,18 @@
 				"18"
 			],
 			"env": "^Shapecss$",
-			"html": "<link[^>]* href=\"[^\"]*shapecss(?:\\.min)?\\.css",
+			"html": [
+				"<link[^>]* href=\"[^\"]*shapecss(?:\\.min)?\\.css",
+				"shapecss\\.(?:min\\.)?(?:[a-f\d]+\\.)?css",
+				"shapecss.*\\.css"
+			]
```
Comment (nekosune) [inline src/apps.json]: I believe a ```,``` is needed here?

Comment (AlexByte) [inline src/apps.json]: @nekosune What makes you think that

Comment (MicroDreamIT) [inline src/apps.json]: no idea :O

**Comment by AlexByte** [comment]
What request?

File: src/apps.json [hunk 2]
```diff
@@ -7154,12 +7154,18 @@
 				"18"
 			],
 			"env": "^Shapecss$",
-			"html": "<link[^>]* href=\"[^\"]*shapecss(?:\\.min)?\\.css",
+			"html": [
+				"<link[^>]* href=\"[^\"]*shapecss(?:\\.min)?\\.css",
+				"shapecss\\.(?:min\\.)?(?:[a-f\d]+\\.)?css",
```
Comment (AlexByte) [inline src/apps.json]: You should write `[a-f\\d]`.

File: src/apps.json [hunk 3]
```diff
@@ -7154,12 +7154,18 @@
...
+				"shapecss.*\\.css"
```
Comment (AlexByte) [inline src/apps.json]: Where is link tag?

File: src/apps.json [hunk 4]
```diff
@@ -7154,12 +7154,18 @@
...
+			]
 			"icon": "Shapecss.svg",
 			"script": [
 				"shapecss[-.]([\\d.]*\\d)[^/]*\\.js\\;version:\\1",
 				"/([\\d.]+)/shapecss(?:\\.min)?\\.js\\;version:\\1",
-				"shapecss.*\\.js"
+				"shapecss.*\\.js",
+				"shapecss\\.(?:min\\.)?(?:[a-f\d]+\\.)?js",
+				"<link[^>]* href=\"[^\"]*shapecss(?:\\.min)?\\.js",
```
Comment (AlexByte) [inline src/apps.json]: You should not use link tag. Еhe comma at the end of the row redundant.

File: src/apps.json (hunk 2 above)
Comment (AlexByte) [inline src/apps.json]: test

**Comment by AlexByte** [comment]
Provide a list of sites other than the official one that use this technology.

File: src/apps.json (hunk 1 above)
Comment (nekosune) [inline src/apps.json]: To make it valid Json, like https://github.com/AliasIO/Wappalyzer/blob/master/src/apps.json#L271
As this is an object by the name of html inside another object alongside other elements, as per the Json specs, as found here, it needs a comma. http://www.json.org

Comment (AlexByte) [inline src/apps.json]: I agree. All errors 
image

**Comment by AliasIO** [comment]
Still broken, closing.

**Pull request closed without merging**